app/views/admin/statistics_announcements/_search_results.html.erb

```diff
@@ -0,0 +1,59 @@
+<p class="govuk-heading-s govuk-!-margin-bottom-3"><%= pluralize(number_with_delimiter(paginator.total_count), "document") %></p>
```

Comment:
It would be semantically better to use the heading component here rather than style a `<p>` element as a heading. Probably the (default) `h2` would be correct and the text size set to the same size as it is here.

Reply:
yeah, I think that's a fair comment as well. Thank you.